dsl/src/main/scala/com/crobox/clickhouse/dsl/column/Magnets.scala

```diff
@@ -28,6 +31,37 @@ trait Magnets {
     val column: TableColumn[C]
   }
 
+  // ComparabeWith trait and Cast case class were members of ComparisonFunctions and TypeCastFunctions trait
+  // respectively. But placing them in the mixin traits causes Scala 3 compiler to crash. Hence, placing these
+  // constructs here is a workaround allowing for the codebase to be compiled with Scala 3.
+  trait ComparableWith[M <: Magnet[_]] {
```

Comment:
In scala 3 the [_] should be rewritten to [?] right?

Reply:
@lwolters Eventually in the future, yes. See https://dotty.epfl.ch/docs/reference/changed-features/wildcards.html but for now it would be too big of a hassle to change